connectivity/tests/host.go

```diff
@@ -193,3 +193,74 @@ func (s *hostToPod) Run(ctx context.Context, t *check.Test) {
 		}
 	}
 }
+
+// HostToWorld sends multiple HTTP requests to ExternalTarget
+// from each node inside the cluster.
+func HostToWorld() check.Scenario {
+	return &hostToWorld{}
+}
+
+// hostToWorld implements a Scenario.
+type hostToWorld struct{}
+
+func (s *hostToWorld) Name() string {
+	return "host-to-world"
+}
+
+func (s *hostToWorld) Run(ctx context.Context, t *check.Test) {
+	extTarget := t.Context().Params().ExternalTarget
+	http := check.HTTPEndpoint(extTarget+"-http", "http://"+extTarget)
+
+	var i int
+	ct := t.Context()
+
+	for _, src := range ct.HostNetNSPodsByNode() {
+		if src.Outside {
+			continue
+		}
+
+		src := src // copy to avoid memory aliasing when using reference
+
+		// With http, over port 80.
+		t.NewAction(s, fmt.Sprintf("http-to-%s-%d", extTarget, i), &src, http, features.IPFamilyAny).Run(func(a *check.Action) {
+			a.ExecInPod(ctx, ct.CurlCommand(http, features.IPFamilyAny))
+		})
+
+		i++
+	}
+}
+
+// HostToWorld2 sends multiple HTTP requests to cilium.io
+// from each node inside the cluster.
+func HostToWorld2() check.Scenario {
```

Comment:
Why is this second scenario required?

Reply:
I need to apply host policy and test that Cilium both allows host egress to the FQDN specified in policy and denies egress to another FQDN. I looked at analogous pod-level tests in connectivity/builder/to_fqdns.go. These use two scenarios (PodToWorld and PodToWorld2) and I copied that.